src/plugins/syntax-highlighting/index.less

```diff
@@ -0,0 +1,9 @@
+*[data-contains-caret="true"] {
```

Comment:
The `*` here (and in the next one) is unnecessary.

Reply:
fixed